ladybug/datatype/temperaturetime.py

```diff
@@ -0,0 +1,66 @@
+# coding=utf-8
+"""Temperature-Time data type."""
+from __future__ import division
+
+from .base import DataTypeBase
+
+
+class TemperatureTime(DataTypeBase):
```

Comment:
Didn't know it is called temperature time! Learned something new today.

Reply:
I did some research and it seems that almost everyone calls it `degree-days` or `degree-hours` but such terminology confuses the name of the units with the name of the data type.  It would be like calling the `Temperature `data type `Degrees`, which will lead to a lot of confusion.  I settled on TemperatureTime after reading the [Wikipedia definition of degree-days](https://en.wikipedia.org/wiki/Degree_day):

A degree day is computed as the integral of a function of **time** that generally varies with **temperature**.